README.rst

```diff
@@ -96,167 +96,10 @@ run Bandit with standard input::
 
     cat examples/imports.py | bandit -
 
-Usage::
```

Comment:
Totally tangential, I think Doug Hellman wrote a plugin to sphinx (or maybe it was Jeff Forcier) that puts the execution output of something in the docs where desired. We could take advantage of that here maybe?

Reply:
We don't _need_ it, though